homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""This component provides support for Stookalert Binary Sensor."""
+from datetime import timedelta
+import logging
+
+import stookalert
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_OFF, STATE_ON
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+CONF_PROVINCE = "province"
+DEFAULT_NAME = "Stookalert"
+PROVINCES = [
+    "drenthe",
+    "flevoland",
+    "friesland",
+    "gelderland",
+    "groningen",
+    "limburg",
+    "noord-brabant",
+    "noord-holland",
+    "overijssel",
+    "utrecht",
+    "zeeland",
+    "zuid-holland",
+]
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_PROVINCE): vol.In(PROVINCES),
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    }
+)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Stookalert binary sensor platform."""
+    add_devices([Stookalert(config)], update_before_add=True)
+
+
+class Stookalert(Entity):
```

Comment:
Please rename this.

```suggestion
class StookAlertBinarySensor(Entity):
```

or 

```suggestion
class StookalertBinarySensor(Entity):
```

Looks like that it should be the second option.